exec/java-exec/src/main/codegen/data/MathFunc.tdd

```diff
@@ -125,7 +125,7 @@ unaryMathFunctions : [
       {input: "UInt8", outputType: "UInt8", castType: "long"}     
      ] 
     },
-    {className: "Mod", funcName: "mod", javaFunc : " % ", types: [
+    {className: "Mod", funcName: "mod", aliasName: "", javaFunc : " % ", types: [
```

Comment:
Same comment as elsewhere... would we want this for other `INT` like data types or does Drill handle that elsewhere?

Reply:
I believe it does handle all widths already.  If you look a full listing of MathFunc.tdd, rather than what this diff reveals, you'll see integer types from 8 bits to 64 bits.